doc/user/zebra.rst

```diff
@@ -351,6 +360,143 @@ default) should the specified gateways not be reachable. Eg:
 
 .. _multicast-rib-commands:
 
+VRF Commands
+============
+
+Currently, the user has the possibility to configure VRFs. VRF is a way to
```

Comment:
Would be great if you could mark this up as:
```
:abbr:`VRF (Virtual Routing and Forwarding)`
```
This way, the parenthesized acronym expansion will show up in a tooltip or a footnote depending on the format. http://www.sphinx-doc.org/en/stable/markup/inline.html#role-abbr

Barring that, it might be good to just explain what the acronym stands for.

Reply:
ok. I do it like this
```
:abbr:`VRF (Virtual Routing and Forwarding)`
======================================
```